### ISSUE TITLE
Sonoff Zigbee Bridge does not report Ikea remote button presses via MQTT

### ISSUE THREAD

**Issue by Carl3k** [arendst/Tasmota#10090]
### PROBLEM DESCRIPTION
After pairing an Ikea remote (type E1524) to a Sonoff Zigbee Bridge ZbStatus2 reports "Unknown device".

### REQUESTED INFORMATION

- [#] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [#] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [#] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [#] Searched the problem in the [forum](https://groups.google.com/d/forum/sonoffusers)
- [#] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [#] Device used: Sonoff Zigbee Bridge
- [#] Tasmota binary firmware version number used: 9.1.0
  - [#] Pre-compiled
- [#] Flashing tools used: esptool.py
- [#] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```
MQT: stat/zbbridge/RESULT = {"NAME":"Generic","GPIO":[1,1,1,1,1,1,1,1,1,1,1,1,1,1],"FLAG":0,"BASE":18}
MQT: stat/zbbridge/RESULT = {"Module":{"75":"Sonoff ZbBridge"}}
MQT: stat/zbbridge/RESULT = {"GPIO0":{"320":"Led_i1"},"GPIO1":{"3552":"Zigbee Tx"},"GPIO2":{"0":"None"},"GPIO3":{"3584":"Zigbee Rx"},"GPIO4":{"5312":"Zigbee Rst"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"640":"I2C SDA"},"GPIO13":{"576":"LedLink_i"},"GPIO14":{"608":"I2C SCL"},"GPIO15":{"0":"None"},"GPIO16":{"32":"Button1"},"GPIO17":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```

```
- [#] Provide the output of this command: `Status 0`:
```
MQT: stat/zbbridge/STATUS = {"Status":{"Module":75,"DeviceName":"zbbridge","FriendlyName":["zbbridge"],"Topic":"zbbridge","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0}}
MQT: stat/zbbridge/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"Power On","Uptime":"0T00:05:01","StartupUTC":"2020-12-08T06:39:15","Sleep":50,"CfgHolder":4617,"BootCount":9,"BCResetTime":"2020-12-04T17:54:55","SaveCount":20,"SaveAddress":"F7000"}}
MQT: stat/zbbridge/STATUS2 = {"StatusFWR":{"Version":"9.1.0(zbbridge)","BuildDateTime":"2020-11-07T12:01:19","Boot":31,"Core":"2_7_4_5","SDK":"2.2.2-dev(38a443e)","CpuFrequency":160,"Hardware":"ESP8266EX","CR":"396/699"}}
MQT: stat/zbbridge/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["MOM",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C8000100060000005A00000000000000","00000000","00006000","00000000"]}}
MQT: stat/zbbridge/STATUS4 = {"StatusMEM":{"ProgramSize":577,"Free":424,"Heap":23,"ProgramFlashSize":1024,"FlashSize":2048,"FlashChipId":"1540A1","FlashFrequency":40,"FlashMode":3,"Features":["00000809","0F1007C6","04400001","00000002","00000000","00000000","00020000","00000000"],"Drivers":"1,2,4,7,9,10,20,23,41","Sensors":""}}
MQT: stat/zbbridge/STATUS5 = {"StatusNET":{"Hostname":"zbbridge","IPAddress":"192.168.2.178","Gateway":"192.168.2.1","Subnetmask":"255.255.255.0","DNSServer":"192.168.2.9","Mac":"98:F4:AB:E2:00:7F","Webserver":2,"WifiConfig":4,"WifiPower":17.0}}
MQT: stat/zbbridge/STATUS6 = {"StatusMQT":{"MqttHost":"house.mom.home","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_E2FF7F","MqttUser":"DVES_USER","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30}}
MQT: stat/zbbridge/STATUS7 = {"StatusTIM":{"UTC":"2020-12-08T06:44:16","Local":"2020-12-08T07:44:16","StartDST":"2020-03-29T02:00:00","EndDST":"2020-10-25T03:00:00","Timezone":"+01:00","Sunrise":"08:31","Sunset":"16:53"}}
MQT: stat/zbbridge/STATUS10 = {"StatusSNS":{"Time":"2020-12-08T07:44:16"}}
MQT: stat/zbbridge/STATUS11 = {"StatusSTS":{"Time":"2020-12-08T07:44:16","Uptime":"0T00:05:01","UptimeSec":301,"Vcc":3.514,"Heap":23,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"MOM","BSSId":"E4:95:6E:12:00:66","Channel":1,"RSSI":58,"Signal":-71,"LinkCount":1,"Downtime":"0T00:00:03"}}}
MQT: tele/zbbridge/STATE = {"Time":"2020-12-08T07:44:24","Uptime":"0T00:05:09","UptimeSec":309,"Vcc":3.523,"Heap":27,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"MOM","BSSId":"E4:95:6E:12:00:66","Channel":1,"RSSI":60,"Signal":-70,"LinkCount":1,"Downtime":"0T00:00:03"}}

```
- [#] Provide the output of the Console log output when you experience your issue; if applicable:
  _(Please use_ `weblog 4` _for more debug information)_
```
CMD: ZbPermitJoin 1
SRC: WebConsole from 192.168.2.164
CMD: Group 0, Index 1, Command "ZBPERMITJOIN", Data "1"
ZIG: ZbEZSPSend 22003C
ZIG: adding packet to_send, to_ack:6, to_send:6, to_end:6
ZIG: ZbEZSPSend 3600FCFF00003600000000000000071E0103073C01
ZIG: adding packet to_send, to_ack:6, to_send:6, to_end:7
MQT: tele/zbbridge/RESULT = {"ZbState":{"Status":21,"Message":"Pairing mode enabled"}}
zigbee.permit_end_time = 800540
MQT: stat/zbbridge/RESULT = {"ZbPermitJoin":"Done"}
ZIG: Something to_send, to_ack:6, to_send:6, to_end:0
ZIG: ZbEZSPSentRaw 6726000122003C
ZIG: {"ZbEZSPReceived2":"77268001220000"}
ZIG: new ack/data received, was 6 now 7
ZIG: freeing packet 6 from memory
ZIG: ZbEZSPSentRaw 80
ZIG: {"ZbEZSPReceived":"220000"}
ZIG: Something to_send, to_ack:7, to_send:7, to_end:0
ZIG: ZbEZSPSentRaw 702700013600FCFF00003600000000000000071E0103073C01
ZIG: {"ZbEZSPReceived2":"002780013600007D"}
ZIG: new ack/data received, was 7 now 0
ZIG: freeing packet 7 from memory
ZIG: ZbEZSPSentRaw 81
ZIG: {"ZbEZSPReceived":"3600007D"}
ZIG: {"ZbEZSPReceived2":"10279001450005000036000000000000007DFF000000FFFF03073C01"}
ZIG: ZbEZSPSentRaw 82
ZIG: {"ZbEZSPReceived":"450005000036000000000000007DFF000000FFFF03073C01"}
ZIG: Internal ZDO message 0x0036 sent from 0x0000 (broadcast)
ZIG: {"ZbEZSPReceived2":"202790013F0006FCFF000036000000000000007D010000"}
ZIG: ZbEZSPSentRaw 83
ZIG: {"ZbEZSPReceived":"3F0006FCFF000036000000000000007D010000"}
WIF: Checking connection...
ZIG: {"ZbEZSPReceived2":"30279401230000001D28BBE7EBFEFF5ECFD004"}
ZIG: ZbEZSPSentRaw 84
ZIG: {"ZbEZSPReceived":"230000001D28BBE7EBFEFF5ECFD004"}
ZIG: {"ZbEZSPReceived2":"4027900124001D28BBE7EBFEFF5ECFD00203FFFF"}
ZIG: ZbEZSPSentRaw 85
ZIG: {"ZbEZSPReceived":"24001D28BBE7EBFEFF5ECFD00203FFFF"}
ZIG: {"ZbEZSPReceived2":"50279001230000011D28BBE7EBFEFF5ECFD004"}
ZIG: ZbEZSPSentRaw 86
ZIG: {"ZbEZSPReceived":"230000011D28BBE7EBFEFF5ECFD004"}
ZIG: {"ZbEZSPReceived2":"6027900124001D28BBE7EBFEFF5ECFD001000000"}
ZIG: ZbEZSPSentRaw 87
ZIG: {"ZbEZSPReceived":"24001D28BBE7EBFEFF5ECFD001000000"}
MQT: tele/zbbridge/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0xD0CF5EFFFEEBE7BB","ShortAddr":"0x281D","ParentNetwork":"0x0000","Status":1,"Decision":0}}
ZIG: ZbFlashStore 010F1D28BBE7EBFEFF5ECFD0000000FF
ZIG: Zigbee Devices Data store in Flash (0x402FF800 - 16 bytes)
ZIG: {"ZbEZSPReceived2":"70279001230000011D28BBE7EBFEFF5ECFD004"}
ZIG: ZbEZSPSentRaw 80
ZIG: {"ZbEZSPReceived":"230000011D28BBE7EBFEFF5ECFD004"}
ZIG: {"ZbEZSPReceived2":"0027900124001D28BBE7EBFEFF5ECFD001000000"}
ZIG: ZbEZSPSentRaw 81
ZIG: {"ZbEZSPReceived":"24001D28BBE7EBFEFF5ECFD001000000"}
MQT: tele/zbbridge/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0xD0CF5EFFFEEBE7BB","ShortAddr":"0x281D","ParentNetwork":"0x0000","Status":1,"Decision":0}}
ZIG: {"ZbEZSPReceived2":"102794016200BBE7EBFEFF5ECFD0"}
ZIG: ZbEZSPSentRaw 82
ZIG: {"ZbEZSPReceived":"6200BBE7EBFEFF5ECFD0"}
ZIG: {"ZbEZSPReceived2":"2027900145000400001300000000040000D5FFDF1D28FFFF0C811D28BBE7EBFEFF5ECFD08002"}
ZIG: ZbEZSPSentRaw 83
ZIG: {"ZbEZSPReceived":"45000400001300000000040000D5FFDF1D28FFFF0C811D28BBE7EBFEFF5ECFD08002"}
MQT: tele/zbbridge/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0xD0CF5EFFFEEBE7BB","ShortAddr":"0x281D","PowerSource":false,"ReceiveWhenIdle":false,"Security":false}}
ZIG: ZbEZSPSend 3400001D2800000500000040010000080103081D28
ZIG: adding packet to_send, to_ack:0, to_send:0, to_end:0
ZIG: Something to_send, to_ack:0, to_send:0, to_end:1
ZIG: ZbEZSPSentRaw 032800013400001D2800000500000040010000080103081D28
ZIG: {"ZbEZSPReceived2":"3128800134000082"}
ZIG: new ack/data received, was 0 now 1
ZIG: freeing packet 0 from memory
ZIG: ZbEZSPSentRaw 84
ZIG: {"ZbEZSPReceived":"34000082"}
ZIG: {"ZbEZSPReceived2":"412890013F0006FDFF00001300000000040000D5000000"}
ZIG: ZbEZSPSentRaw 85
ZIG: {"ZbEZSPReceived":"3F0006FDFF00001300000000040000D5000000"}
ZIG: {"ZbEZSPReceived2":"5128900145000000000580000040010000D6FFDF1D28FFFF0608001D28010102"}
ZIG: ZbEZSPSentRaw 86
ZIG: {"ZbEZSPReceived":"45000000000580000040010000D6FFDF1D28FFFF0608001D28010102"}
MQT: tele/zbbridge/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x01"]}}
ZIG: ZbEZSPSend 3400001D280401000001014001000001010700010004000500
ZIG: adding packet to_send, to_ack:1, to_send:1, to_end:1
ZIG: Something to_send, to_ack:1, to_send:1, to_end:2
ZIG: ZbEZSPSentRaw 162900013400001D280401000001014001000001010700010004000500
ZIG: {"ZbEZSPReceived2":"612890013F00001D280000050000004001000082010000"}
ZIG: ZbEZSPSentRaw 87
ZIG: {"ZbEZSPReceived":"3F00001D280000050000004001000082010000"}
ZIG: {"ZbEZSPReceived2":"7229800134000083"}
ZIG: new ack/data received, was 1 now 2
ZIG: freeing packet 1 from memory
ZIG: ZbEZSPSentRaw 80
ZIG: {"ZbEZSPReceived":"34000083"}
ZIG: {"ZbEZSPReceived2":"0229900145000004010000010140010000D8FFE51D28FFFF31180101040000420E494B4541206F662053776564656E0500004216545241444652492072656D6F746520636F6E74726F6C02"}
ZIG: ZbEZSPSentRaw 81
ZIG: {"ZbEZSPReceived":"45000004010000010140010000D8FFE51D28FFFF31180101040000420E494B4541206F662053776564656E0500004216545241444652492072656D6F746520636F6E74726F6C02"}
ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":0,"srcaddr":"0x281D","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":157,"securityuse":0,"seqnumber":216,"fc":"0x18","manuf":"0x0000","transact":1,"cmdid":"0x01","payload":"040000420E494B4541206F662053776564656E0500004216545241444652492072656D6F746520636F6E74726F6C"}}
ZIG: ZbZCLRawReceived: {"0x281D":{"0000/0004":"IKEA of Sweden","0000/0005":"TRADFRI remote control","Endpoint":1,"LinkQuality":157}}
MQT: tele/zbbridge/SENSOR = {"ZbReceived":{"0x281D":{"Device":"0x281D","Manufacturer":"IKEA of Sweden","ModelId":"TRADFRI remote control","Endpoint":1,"LinkQuality":157}}}
ZIG: {"ZbEZSPReceived2":"122990013F00001D280401000001014001000083010000"}
ZIG: ZbEZSPSentRaw 82
ZIG: {"ZbEZSPReceived":"3F00001D280401000001014001000083010000"}
ZIG: {"ZbEZSPReceived2":"222990013F0003FDFF0401060001FF00012B3BD7000000"}
ZIG: ZbEZSPSentRaw 83
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BD7000000"}
ZIG: Sniffing group 0x3B2B
ZIG: ZbEZSPSend 3400001D2800000400000040010000090104091D2801
ZIG: adding packet to_send, to_ack:2, to_send:2, to_end:2
ZIG: Something to_send, to_ack:2, to_send:2, to_end:3
ZIG: ZbEZSPSentRaw 232A00013400001D2800000400000040010000090104091D2801
ZIG: {"ZbEZSPReceived2":"332A800134000084"}
ZIG: new ack/data received, was 2 now 3
ZIG: freeing packet 2 from memory
ZIG: ZbEZSPSentRaw 84
ZIG: {"ZbEZSPReceived":"34000084"}
ZIG: {"ZbEZSPReceived2":"432A900145000000000480000040010000D9FFE61D28FFFF2709001D2822015EC0300802060000010003000900050B001007030004000500060008001900001002"}
ZIG: ZbEZSPSentRaw 85
ZIG: {"ZbEZSPReceived":"45000000000480000040010000D9FFE61D28FFFF2709001D2822015EC0300802060000010003000900050B001007030004000500060008001900001002"}
MQT: tele/zbbridge/RESULT = {"ZbState":{"Status":33,"Endpoint":"0x01","ProfileId":"0xC05E","DeviceId":"0x0830","DeviceVersion":2"InClusters":["0x0000","0x0001","0x0003","0x0009","0x0B05","0x1000"],"OutClusters":["0x0003","0x0004","0x0005","0x0006","0x0008","0x0019","0x1000"]}}
ZIG: {"ZbEZSPReceived2":"532A90013F00001D280000040000004001000084010000"}
ZIG: ZbEZSPSentRaw 86
ZIG: {"ZbEZSPReceived":"3F00001D280000040000004001000084010000"}
ZIG: {"ZbEZSPReceived2":"632A90013F0003FDFF0401060001FF00012B3BDA000000"}
ZIG: ZbEZSPSentRaw 87
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BDA000000"}
ZIG: Sniffing group 0x3B2B
ZIG: ZbFlashStore 01351D28BBE7EBFEFF5ECFD0545241444652492072656D6F746520636F6E74726F6C00494B4541206F662053776564656E000001FFFF
ZIG: Zigbee Devices Data store in Flash (0x402FF800 - 54 bytes)
ZIG: auto-bind `ZbBind {"Device":"0x281D","Endpoint":1,"Cluster":"0x0001"}`
ZIG: ZbEZSPSend 3400001D28000021000000400100000A01160ABBE7EBFEFF5ECFD0010100034E1020FEFF23A46001
ZIG: adding packet to_send, to_ack:3, to_send:3, to_end:3
ZIG: Something to_send, to_ack:3, to_send:3, to_end:4
ZIG: ZbEZSPSentRaw 372B00013400001D28000021000000400100000A01160ABBE7EBFEFF5ECFD0010100034E1020FEFF23A46001
ZIG: {"ZbEZSPReceived2":"742B800134000085"}
ZIG: new ack/data received, was 3 now 4
ZIG: freeing packet 3 from memory
ZIG: ZbEZSPSentRaw 80
ZIG: {"ZbEZSPReceived":"34000085"}
ZIG: {"ZbEZSPReceived2":"042B900145000000002180000040010000DBFFE41D28FFFF020A0002"}
ZIG: ZbEZSPSentRaw 81
ZIG: {"ZbEZSPReceived":"45000000002180000040010000DBFFE41D28FFFF020A0002"}
MQT: tele/zbbridge/RESULT = {"ZbBind":{"Device":"0x281D","Status":0,"StatusMessage":"SUCCESS"}}
ZIG: {"ZbEZSPReceived2":"142B90013F00001D280000210000004001000085010000"}
ZIG: ZbEZSPSentRaw 82
ZIG: {"ZbEZSPReceived":"3F00001D280000210000004001000085010000"}
WIF: Checking connection...
ZIG: auto-bind `ZbBind {"Device":"0x281D","Endpoint":1,"Cluster":"0x0006"}`
ZIG: ZbEZSPSend 3400001D28000021000000400100000B01160BBBE7EBFEFF5ECFD0010600034E1020FEFF23A46001
ZIG: adding packet to_send, to_ack:4, to_send:4, to_end:4
ZIG: Something to_send, to_ack:4, to_send:4, to_end:5
ZIG: ZbEZSPSentRaw 422C00013400001D28000021000000400100000B01160BBBE7EBFEFF5ECFD0010600034E1020FEFF23A46001
ZIG: {"ZbEZSPReceived2":"252C800134000086"}
ZIG: new ack/data received, was 4 now 5
ZIG: freeing packet 4 from memory
ZIG: ZbEZSPSentRaw 83
ZIG: {"ZbEZSPReceived":"34000086"}
ZIG: {"ZbEZSPReceived2":"352C900145000000002180000040010000DCFFE41D28FFFF020B0002"}
ZIG: ZbEZSPSentRaw 84
ZIG: {"ZbEZSPReceived":"45000000002180000040010000DCFFE41D28FFFF020B0002"}
MQT: tele/zbbridge/RESULT = {"ZbBind":{"Device":"0x281D","Status":0,"StatusMessage":"SUCCESS"}}
ZIG: {"ZbEZSPReceived2":"452C90013F00001D280000210000004001000086010000"}
ZIG: ZbEZSPSentRaw 85
ZIG: {"ZbEZSPReceived":"3F00001D280000210000004001000086010000"}
ZIG: auto-bind `ZbBind {"Device":"0x281D","Endpoint":1,"Cluster":"0x0008"}`
ZIG: ZbEZSPSend 3400001D28000021000000400100000C01160CBBE7EBFEFF5ECFD0010800034E1020FEFF23A46001
ZIG: adding packet to_send, to_ack:5, to_send:5, to_end:5
ZIG: Something to_send, to_ack:5, to_send:5, to_end:6
ZIG: ZbEZSPSentRaw 552D00013400001D28000021000000400100000C01160CBBE7EBFEFF5ECFD0010800034E1020FEFF23A46001
ZIG: {"ZbEZSPReceived2":"562D800134000087"}
ZIG: new ack/data received, was 5 now 6
ZIG: freeing packet 5 from memory
ZIG: ZbEZSPSentRaw 86
ZIG: {"ZbEZSPReceived":"34000087"}
ZIG: {"ZbEZSPReceived2":"662D900145000000002180000040010000DEFFE11D28FFFF020C0002"}
ZIG: ZbEZSPSentRaw 87
ZIG: {"ZbEZSPReceived":"45000000002180000040010000DEFFE11D28FFFF020C0002"}
MQT: tele/zbbridge/RESULT = {"ZbBind":{"Device":"0x281D","Status":0,"StatusMessage":"SUCCESS"}}
ZIG: {"ZbEZSPReceived2":"762D90013F00001D280000210000004001000087010000"}
ZIG: ZbEZSPSentRaw 80
ZIG: {"ZbEZSPReceived":"3F00001D280000210000004001000087010000"}
ZIG: {"ZbEZSPReceived2":"062D90013F0003FDFF0401060001FF00012B3BDD000000"}
ZIG: ZbEZSPSentRaw 81
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BDD000000"}
ZIG: Sniffing group 0x3B2B
ZIG: auto-bind `ZbSend {"Device":"0x281D","Config":{"BatteryVoltage":{"MinInterval":3600,"MaxInterval":14400,"ReportableChange":0.1},"BatteryPercentage":{"MinInterval":3600,"MaxInterval":14400,"ReportableChange":1}}}`
ZIG: ZbEZSPSend 3400001D280401010001014001000002011510020600200020100E40380100210020100E403802
ZIG: adding packet to_send, to_ack:6, to_send:6, to_end:6
ZIG: Something to_send, to_ack:6, to_send:6, to_end:7
ZIG: ZbEZSPSentRaw 612E00013400001D280401010001014001000002011510020600200020100E40380100210020100E403802
ZIG: {"ZbEZSPReceived2":"172E800134000088"}
ZIG: new ack/data received, was 6 now 7
ZIG: freeing packet 6 from memory
ZIG: ZbEZSPSentRaw 82
ZIG: {"ZbEZSPReceived":"34000088"}
ZIG: {"ZbEZSPReceived2":"272E900145000004010100010140010000DFFFE41D28FFFF040802070002"}
ZIG: ZbEZSPSentRaw 83
ZIG: {"ZbEZSPReceived":"45000004010100010140010000DFFFE41D28FFFF040802070002"}
ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":1,"srcaddr":"0x281D","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":154,"securityuse":0,"seqnumber":223,"fc":"0x08","manuf":"0x0000","transact":2,"cmdid":"0x07","payload":"00"}}
ZIG: ZbZCLRawReceived: {"0x281D":{"ConfigResponse":{},"Endpoint":1,"LinkQuality":154}}
MQT: tele/zbbridge/SENSOR = {"ZbReceived":{"0x281D":{"Device":"0x281D","ConfigResponse":{},"Endpoint":1,"LinkQuality":154}}}
ZIG: {"ZbEZSPReceived2":"372E90013F00001D280401010001014001000088010000"}
ZIG: ZbEZSPSentRaw 84
ZIG: {"ZbEZSPReceived":"3F00001D280401010001014001000088010000"}
ZIG: {"ZbEZSPReceived2":"472E90013F0003FDFF0401060001FF00012B3BE0000000"}
ZIG: ZbEZSPSentRaw 85
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BE0000000"}
ZIG: Sniffing group 0x3B2B
WIF: Checking connection...
ZIG: {"ZbEZSPReceived2":"572E90013F0003FDFF0401060001FF00012B3BE1000000"}
ZIG: ZbEZSPSentRaw 86
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BE1000000"}
ZIG: Sniffing group 0x3B2B
MQT: tele/zbbridge/RESULT = {"ZbState":{"Status":20,"Message":"Pairing mode disabled"}}

```

### TO REPRODUCE
_Steps to reproduce the behavior:_
- ZbPermitJoin 1
- pressing pairing button on remote 4 times within 5 seconds, red LED on remote front blinks
- ZbStatus2 shows MQT: stat/zbbridge/RESULT = {"ZbStatus":"Unknown device"}

### EXPECTED BEHAVIOUR
The remote is recognised by the bridge, button presses are reported.

### SCREENSHOTS
Both devices do not have a screen.

### ADDITIONAL CONTEXT
Bridge has been flashed based on instructions from https://zigbee.blakadder.com/Sonoff_ZBBridge.html .

**Comment by s-hadinger** [comment]
Type `ZbStatus2 0x281D` or `ZbStatus` to get the whole list of devices

**Comment by Carl3k** [comment]
That indeed shows some details:
```
CMD: ZbStatus
MQT: stat/zbbridge/RESULT = {"ZbStatus1":[{"Device":"0x281D"}]}

CMD: ZbStatus2 0x281D
MQT: stat/zbbridge/RESULT = {"ZbStatus2":[{"Device":"0x281D","IEEEAddr":"0xD0CF5EFFFEEBE7BB","ModelId":"TRADFRI remote control","Manufacturer":"IKEA of Sweden","Endpoints":[1],"Config":[]}]}
```

Problem is that pressing buttons on the remote only activates the Zigbee LED on the bridge. No MQTT message is generated. Has something else to be done besides pairing?

**Comment by s-hadinger** [comment]
No, from your logs the binding went ok.

Please reach out via Discord.

**Comment by Carl3k** [comment]
Will ask on Discord as adviced.

For reference, these are log entries for button presses after setting WebLog to 4:


center button
```
ZIG: {"ZbEZSPReceived2":"233A90013F0003FDFF0401060001FF00012B3BFB000000"}
ZIG: ZbEZSPSentRaw 83
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BFB000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"333A90013F0003FDFF0401060001FF00012B3BFC000000"}
ZIG: ZbEZSPSentRaw 84
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BFC000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"433A90013F0003FDFF0401060001FF00012B3BFD000000"}
ZIG: ZbEZSPSentRaw 85
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401060001FF00012B3BFD000000"}
ZIG: Sniffing group 0x3B2B
```


top button
```
ZIG: {"ZbEZSPReceived2":"533A90013F0003FDFF0401080001FF00012B3BFE000000"}
ZIG: ZbEZSPSentRaw 86
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401080001FF00012B3BFE000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"633A90013F0003FDFF0401080001FF00012B3BFF000000"}
ZIG: ZbEZSPSentRaw 87
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401080001FF00012B3BFF000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"733A90013F0003FDFF0401080001FF00012B3B00000000"}
ZIG: ZbEZSPSentRaw 80
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401080001FF00012B3B00000000"}
ZIG: Sniffing group 0x3B2B
```


bottom button
```
ZIG: {"ZbEZSPReceived2":"033A90013F0003FDFF0401080001FF00012B3B01000000"}
ZIG: ZbEZSPSentRaw 81
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401080001FF00012B3B01000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"133A90013F0003FDFF0401080001FF00012B3B02000000"}
ZIG: ZbEZSPSentRaw 82
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401080001FF00012B3B02000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"233A90013F0003FDFF0401080001FF00012B3B03000000"}
ZIG: ZbEZSPSentRaw 83
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401080001FF00012B3B03000000"}
ZIG: Sniffing group 0x3B2B
```


left button
```
ZIG: {"ZbEZSPReceived2":"333A90013F0003FDFF0401050001FF00012B3B04000000"}
ZIG: ZbEZSPSentRaw 84
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401050001FF00012B3B04000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"433A90013F0003FDFF0401050001FF00012B3B05000000"}
ZIG: ZbEZSPSentRaw 85
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401050001FF00012B3B05000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"533A90013F0003FDFF0401050001FF00012B3B06000000"}
ZIG: ZbEZSPSentRaw 86
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401050001FF00012B3B06000000"}
ZIG: Sniffing group 0x3B2B
```


right button
```
ZIG: {"ZbEZSPReceived2":"633A90013F0003FDFF0401050001FF00012B3B07000000"}
ZIG: ZbEZSPSentRaw 87
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401050001FF00012B3B07000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"733A90013F0003FDFF0401050001FF00012B3B08000000"}
ZIG: ZbEZSPSentRaw 80
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401050001FF00012B3B08000000"}
ZIG: Sniffing group 0x3B2B
ZIG: {"ZbEZSPReceived2":"033A90013F0003FDFF0401050001FF00012B3B09000000"}
ZIG: ZbEZSPSentRaw 81
ZIG: {"ZbEZSPReceived":"3F0003FDFF0401050001FF00012B3B09000000"}
ZIG: Sniffing group 0x3B2B
```

**Comment by s-hadinger** [comment]
From your logs it looks like you have an old firmware in your device where group addresses are fixed in advance.

Your group is 0x3B2B or 15147.

Type `ZbListen 15147` and you will see the events coming in MQTT. Be aware that you need to put a rule to run `ZbListen1 15147` at each restart:

Example:
```
Rule1 on zbstate#status==0 do backlog zblisten1 15147 endon 
Rule1 1
```

**Comment by Carl3k** [comment]
Brilliant, thanks a lot for your analysis. Setting ZbListen indeed works, MQTT messages are created.

I bought the remote a long time ago. Now after I got the Sonoff Zigbee Bridge it was the only device to test functionality of the setup. It is very likely that firmware in the remote is old.

Is there any other way to do a firmware update on the device than finding an Ikea bridge or someone with a Zigbee2mqtt setup?

**Comment by s-hadinger** [comment]
Software update of IKEA devices works with an IKEA gateway. But I believe you can also use Zigbee2MQTT.

OTA Zigbee update is not supported by Tasmota and will probably never be. ESP8266 is too limited for this.

**Comment by Carl3k** [comment]
Thanks a lot for confirming the options to update Ikea devices. You are a true Zigbee expert. :)

**Comment by JoeWiseman** [comment]
Sorry to open this issue again ....
I've got the same problems described above ... So I thought about buying a IKEA-gateway to upgrade my remotes. All went fine with that, so that the firmware has been updated today successfully.
When connecting the remote to the ZBbridge (which was flashed as told at: https://zigbee.blakadder.com/Sonoff_ZBBridge.html eith the lates firmware: ncp-uart-sw_6.7.8_115200.ota and works allready fine with Tuya temperature sensors) the same problem described above appears. Of course the same workaround is helping.... But I thought when uprading firmware it should work without the listen-option....
Is there any way to implement this, so that in future versions it will be possible to use the IKEA tradfri without  that workaround?
Regards
Joe
